### PR TITLE
Return error info when input irregular or illegal para.

### PR DESCRIPTION
**Pull request by bliuredhat**
JIRA: PDC-998

### CODE REVIEW

**Comment by bliuredhat** [comment]
@lubomir , @simozhan  and @ycheng-aa ; please review this patch.

File: bin/pdc_client [hunk 1]
```diff
@@ -50,6 +50,9 @@ class RequestMethod(object):
 
     @debug_request
     def get(self, data):
+        if isinstance(data, list):
+            print 'Parameter %s is error format %s, please input right format.' % (data, type(data))
```
Comment (ycheng-aa) [inline bin/pdc_client]: Should the data be a dict? If so, maybe the condition could be 
if not isinstance(data, dict):
    .........................

Should we also point out what is the correct format in error message?

**Comment by ycheng-aa** [comment]
I also prefer the commit message more specific rather than the PDC number.

File: bin/pdc_client (hunk 1 above)
Comment (bliuredhat) [inline bin/pdc_client]: yes,  your ideal is right.

**Pull request merged**